test/clubbing_top_level.txt

```diff
@@ -0,0 +1 @@
+<table border="1"><tr><th>aaa</th><th>ccc</th></tr><tr><td>1</td><td>2</td></tr><tr><td>bbb</td><td>2</td></tr></table>
```

Comment:
Can you check again with latest version? I think this is fixed now and I also added a test for this.